Gems/Multiplayer/Code/Include/Multiplayer/IMultiplayerDebug.h

```diff
@@ -7,9 +7,177 @@
  */
 
 #pragma once
+#include <AzCore/Math/MathStringConversions.h>
 
 namespace Multiplayer
 {
+    enum class MultiplayerAuditCategory
+    {
+        MP_AUDIT_DESYNC,
```

Comment:
Since this is an enum class, and you have to scope usage in any case, you can go with more readable names:

```
enum class AuditCategory
{
   Desync, 
   Input,
   Event
}
```

and then you have `Multiplayer::AuditCategory::Input` etc.. rather than triplicate the multiplayer bit, duplicate the audit bit, and use macro name formatting, `Multiplayer::MultiplayerAuditCategory::MP_AUDIT_INPUT`